lib/manageiq/util/ftp_lib.rb

```diff
@@ -0,0 +1,71 @@
+require 'net/ftp'
```

Comment:
This should just live in /lib, IMO

Reply:
Not opposed to it, just was putting it in close proximity to the script that was using it, and also the namespace doesn't hurt avoiding potential conflicts (`FtpLib` is not my most creative name...).